byron/ledger/formal-spec/properties.tex

```diff
@@ -222,12 +222,12 @@ \subsubsection{PVBUMP Properties}
 available, the system remains in the same state it started with.
 
 \begin{property}[PVBUMP no change on empty future adoptions]\label{PVBUMP-empty-future-adopt}
-  For all $\var{s_n}$, $\var{fads}$, $\var{pv}$ and $\var{pps}$ such
-  that:
+  \textnormal{For all $\var{s_n}$, $\var{fads}$, $\var{pv}$ and $\var{pps}$ such
+  that:}
```

Comment:
I wouldn't use `textnormal` and just leave the text in italics. But if you prefer it this way, please make sure that the rest of the document remains consistent.

Reply:
This is just the general idea of systems like LaTeX, where rendering and content are separated. That's why I went for `\textnormal` instead of explicitly saying the text should be e.g. in italics.

Sure, I can update prose in other properties to use `\textnormal`.

Reply:
We don't say *explicitly* that text should be in italics! This is made implicitly! And before you do this, please check how this is done in academic publications that use latex, as far as I remember they use `newtheorem` and italics: see for instance:

- https://www.microsoft.com/en-us/research/uploads/prod/2019/03/eta.pdf
- https://www.microsoft.com/en-us/research/wp-content/uploads/2016/11/levity-pldi17.pdf
- http://homepages.inf.ed.ac.uk/wadler/papers/cochis/cochis.pdf

Also note that if you change the style of `newtheorem`'s here, we might want to do that in the shelley spec as well.

Reply:
I appreciate your effort to link to PDFs, but can you point to LaTeX source examples please?

It's not that I changed the style of `newtheorem` environment; instead, in those properties I wrote I've put prose in `\textnormal`.

I'm new to using the `property` environment so I'd be happy to learn what is preferred over what and for what reason!

Reply:
> I appreciate your effort to link to PDFs, but can you point to LaTeX source examples please?

Oh, sorry for the confusion. In the examples I sent, you can see that the text in the theorems and properties is written in italics. So what I wanted to show with the examples above was just that (no source needed ;) ).

 > It's not that I changed the style of newtheorem environment; instead, in those properties I wrote I've put prose in \textnormal.

And that's exactly what I'm arguing here, I'm used to having the text inside new theorem be in italics, which I think makes sense since:

- we don't have to define our custom format 
- we don't have to remember to put the text in the definition inside a `textnormal` macro